src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        this.side = side;
+        setFigureType("square");
+        setColor(color);
+        calculateArea();
+    }
+
+    @Override
+    public void calculateArea() {
+        setArea(side * side);
+    }
+
+    @Override
+    public String toString() {
+        return super.toString()
+                + getFigureType()
+                + ", area: "
+                + getArea()
+                + " sq.units"
+                + ", side: "
+                + side
+                + " units, color: "
+                + getColor();
```

Comment:
From the requirements:

```
To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this)
```

So, create an interface with one abstract method `draw()` and implement it in your `Figure`.